docs/Configuration/yaml-files.md

```diff
@@ -260,6 +260,10 @@ controls:
     bootstrap_package: https://example.org/bootstrap_package.pkg
     enable_end_user_authentication: true
     macos_setup_assistant: ../lib/dep-profile.json
+    script: ../lib/macos-setup-script.sh
+    software:
+      - app_store_id: '1091189122'
+      - package_path: ../lib/software/adobe-acrobat.software.yml
```

Comment:
I think we went w/ this to be consistent w/ how we reference software in the install software policy automations [here](https://fleetdm.com/docs/configuration/yaml-files#separate-file):

![Screenshot 2024-10-03 at 4 58 28 PM](https://github.com/user-attachments/assets/fd2aeba8-7de4-4c15-8672-1a3bc231158a)

But I see that we'd now be using a generic `path` for some items and a more specific `package_path` for software.

I'm not sure why we went w/ `package_path` there^ instead of `path`. @marko-lisica do you know why?

Reply:
We wanted to be more explicit so users know it's pakage. Later we might support App Store apps under `install_software` in policies. It might be confusing if it's only `path`.

I know we use `path` in other places, but it's usually under the key that describes for what we use `path`. I'm ok with `path` if you think it's clear.

Reply:
Thanks Marko!

I'm ok w/ either. I think up to @rachaelshaw.